Example/PaymentSheet Example/PaymentSheet Example/CustomerSheetTestPlaygroundController.swift

```diff
@@ -129,20 +129,29 @@ class CustomerSheetTestPlaygroundController: ObservableObject {
         return configuration
     }
 
-    func customerAdapter(customerId: String, ephemeralKey: String, configuration: CustomerSheet.Configuration) -> StripeCustomerAdapter {
+    func customerAdapter(customerId: String, ephemeralKey: String?, customerSessionClientSecret: String?, configuration: CustomerSheet.Configuration) -> StripeCustomerAdapter {
         let customerAdapter: StripeCustomerAdapter
         switch settings.paymentMethodMode {
         case .setupIntent:
-            customerAdapter = StripeCustomerAdapter(customerEphemeralKeyProvider: {
-                // This should be a block that fetches this from your server
-                .init(customerId: customerId, ephemeralKeySecret: ephemeralKey)
-            }, setupIntentClientSecretProvider: {
-                return try await self.backend.createSetupIntent(customerId: customerId, merchantCountryCode: self.settings.merchantCountryCode.rawValue)
-            })
+            if let customerSessionClientSecret {
+                customerAdapter = StripeCustomerAdapter(customerSessionClientSecretProvider: {
+                    // This should be a block that fetches this from your server
+                    return .init(customerId: customerId, clientSecret: customerSessionClientSecret)
+                }, setupIntentClientSecretProvider: {
+                    return try await self.backend.createSetupIntent(customerId: customerId, merchantCountryCode: self.settings.merchantCountryCode.rawValue)
+                })
+            } else {
+                customerAdapter = StripeCustomerAdapter(customerEphemeralKeyProvider: {
+                    // This should be a block that fetches this from your server
+                    return .init(customerId: customerId, ephemeralKeySecret: ephemeralKey!)
```

Comment:
There is an issue where -- when customer ephemeral key is selected with customer session.